.github/workflows/ruby_tests.yml

```diff
@@ -83,7 +83,7 @@ jobs:
         node-version: [12]
         include:
           - foreman-core-branch: puppet_extraction
```

Comment:
Do our branches differs?
I'm trying to keep it up to date by constatnt rebases and I'll include any changes from reviews also in that branch.
But hopefully we soon can change this to develop, so I dont mind much :)

Reply:
I temporarily changed to my branch where I extracted the graphql stuff and rebased with develop. Before this PR is merged we should update your branch, but let's wait to see if the tests pass

Reply:
opened https://github.com/ezr-ondrej/foreman/pull/66